components/sections/Pricing.tsx

```diff
@@ -0,0 +1,131 @@
+import { ReactElement } from "react";
+import { Box, Flex, Grid, Text } from "@chakra-ui/core";
+import styled from "@emotion/styled";
+import PricingBanner from "../svgs/pricing-banner.svg";
+import PricingCard from "@/components/PricingCard";
+import theme from "@/src/theme";
+import { LocationHashEnum } from "@/src/enums";
+import RegisterButton from "@/components/RegisterButton";
+
+type PricingInfo = {
+  title: string;
+  price: string;
+  description: string;
+  benefits: string[];
+};
+
+const freeTierInfo: PricingInfo = {
+  title: "Free",
+  price: "$0",
+  description: "Get a complete experience.",
+  benefits: [
+    "Access to our virtual conference platform Hopin.to",
+    "All talks from speakers",
+    "Networking in our virtual career fair with the sponsors",
+    "Social events",
+    "Workshops hosted by our sponsors",
+    "And more to come!",
+  ],
+};
+
+const vipTierInfo: PricingInfo = {
+  title: "VIP",
+  price: "$25",
+  description: "Get an exclusive experience.",
+  benefits: [
+    "All free tier benefits",
+    "CUSEC 2021 merch",
+    "Exclusive and priority networking",
+    "Access to a private VIP communication platform with the sponsors and speakers",
+    "VIP Q&A events at the end of each night with speakers and sponsors",
+    "And more to come!",
+  ],
+};
+
+const monoGridArea = "I_LOVE_CUSEC";
```

Comment:
haha

Reply:
I thought this was more fun than something typical when choosing an arbitrary string for the grid layout :stuck_out_tongue:

Reply:
For sure!